bot/commands/rss_feeds/rssfeeds.js

```diff
@@ -0,0 +1,29 @@
+const {Command} = require('yuuko');
+
+module.exports = new Command('rssfeeds', async (message, args, context) => {
+	const {db} = context;
+	const collection = db.collection('rssFeeds');
+	let str = '';
+	const feeds = await collection.find().toArray();
+
+	if (feeds.length === 0) {
+		message.channel.createMessage('No feeds found.').catch(() => { });
+		return;
+	}
+
+	for (const feed of feeds) {
+		const channelName = context.client.getChannel(feed.channelId).name;
+		str += `Feed: **${feed.rssFeedName}**\nChannel: **${channelName}**\nURL: ${feed.rssFeedUrl} \n \n`;
```

Comment:
Feed name and channel name should both be passed through `escape()` like above, URLs don't need to be but should be wrapped in `<>` to suppress any previews that might be generated otherwise.